test/provisioner/provision_ibmcloud.go

```diff
@@ -294,43 +321,83 @@ func (p *IBMCloudProvisioner) CreateCluster(ctx context.Context, cfg *envconf.Co
 		},
 	}
 	target := containerv2.ClusterTargetHeader{}
-	ibmcloudTrace(fmt.Sprintf("Creating cluster %s.\n", IBMCloudProps.ClusterName))
+	log.Infof("Creating cluster %s.\n", IBMCloudProps.ClusterName)
 	_, err := IBMCloudProps.ClusterAPI.Create(clusterInfo, target)
 	if err != nil {
 		return err
 	}
 
-	// TODO, wait cluter ready, it takes long time
+	clusterReady := false
+	waitMinutes := 50
+	log.Infof("Waiting for cluster %s to be available.\n", IBMCloudProps.ClusterName)
+	for i := 0; i <= waitMinutes; i++ {
+		ready, err := isClusterReady(IBMCloudProps.ClusterName)
+		if err != nil {
+			log.Infof("Err %s happened when retrieve cluster, try again...\n", err)
+			continue
+		}
+		if ready {
+			log.Infof("Cluster %s is available.\n", IBMCloudProps.ClusterName)
+			clusterReady = true
+			break
+		}
+		log.Infof("Waited %d minutes...\n", i)
+
+		time.Sleep(60 * time.Second)
+	}
+
+	if !clusterReady {
+		return fmt.Errorf("Cluster %s was created but not ready in %d minutes.\n", IBMCloudProps.ClusterName, waitMinutes)
+	}
 	return nil
 }
 
 func (p *IBMCloudProvisioner) CreateVPC(ctx context.Context, cfg *envconf.Config) error {
-	ibmcloudTrace("CreateVPC()")
+	log.Info("CreateVPC()")
 	return createVpcImpl()
 }
 
 func (p *IBMCloudProvisioner) DeleteCluster(ctx context.Context, cfg *envconf.Config) error {
-	ibmcloudTrace("DeleteCluster()")
+	log.Info("DeleteCluster()")
```

Comment:
I think this could have the `Debug` level?

Reply:
Agree